examples/federated-diffusion/client.py

```diff
@@ -0,0 +1,146 @@
+import json
+import pickle
+from collections import OrderedDict
+
+import torch
+from centralized import get_model, train, validate
+from config import PARAMS
+from data import load_datasets
+
+import flwr as fl
+
+DEVICE = torch.device(PARAMS.device)  # Try "cuda" to train on GPU
```

Comment:
Can we not use global variables for simulation? the reason is that this will interfere with whatever `client_resources` are passed to `start_simulation`. Clients can "discover" the device they have access to by doing for instance:

```python
device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

either inside the client object or inside `client_fn`